src/main/java/services/UserService.java

```diff
@@ -0,0 +1,13 @@
+package services;
+
+import db.UserStorage;
+import java.util.List;
+import model.Bet;
+
+public class UserService {
```

Comment:
if you have User service so let's make each call to userDao  from consoleHandler through it. And create the same service for Bet

Reply:
done